usaspending_api/accounts/models.py

```diff
@@ -165,7 +165,20 @@ class AppropriationAccountBalances(DataSourceTrackedModel):
     certified_date = models.DateField(blank=True, null=True)
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
+    final_of_fy = models.BooleanField(blank=False, null=False, default=False, db_index=True)
 
     class Meta:
         managed = True
         db_table = 'appropriation_account_balances'
+
+    @classmethod
+    def populate_final_of_fy(cls):
+        sql = """   UPDATE appropriation_account_balances
```

Comment:
This one might be do-able via ORM. The particular thing I've found that the ORM doesn't do is `UPDATE` via a table join. Agreed that we should have a shared understanding of when we can/should drop to raw SQL, but would disagree that we have SQL creep.

Aside from migrations--where I think it's ok to be less stringent about SQL vs ORM--the only other place I see raw SQL in the code base is the award helpers (and the currently PR'd federal accounts helpers). Both of those fall into the "update via table join pattern" that isn't supported.